tests/build/test_build_batch.py

```diff
@@ -0,0 +1,37 @@
+from NIPTool.build.document import build_batch
+import pytest
+
```

Comment:
I think you might add a parametrized test for 'convert(...)' that all attributes that you want None-ified gets None when empty. 

I think you might add a parametrized test for 'build_batch(...)' that all attributes that you want included are included (assuming they have a value)

Reply:
👍